### ISSUE TITLE
VScrollBarAccessibleObject_Ctor_Default failing intermittently in CI

### ISSUE THREAD

**Issue by JeremyKuhne** [dotnet/winforms#3856]
System.Windows.Forms.Tests.VScrollBar_VScrollBarAccessibleObjectTests.VScrollBarAccessibleObject_Ctor_Default(createControl: True, accessibleRole: ScrollBar)

```
Assert.Equal() Failure\r\nExpected: ScrollBar\r\nActual:   Client

  at System.Windows.Forms.Tests.VScrollBar_VScrollBarAccessibleObjectTests.VScrollBarAccessibleObject_Ctor_Default(Boolean createControl, AccessibleRole accessibleRole) in F:\workspace\_work\1\s\src\System.Windows.Forms\tests\UnitTests\VScrollBar.VScrollBarAccessibleObjectTests.cs:line 33
```

https://dev.azure.com/dnceng/public/_build/results?buildId=806036&view=ms.vss-test-web.build-test-results-tab&runId=25536694&resultId=102373&paneView=debug

**Comment by JeremyKuhne** [comment]
Failed again: https://dev.azure.com/dnceng/public/_build/results?buildId=828192&view=ms.vss-test-web.build-test-results-tab&runId=26395620&resultId=105198&paneView=debug

**Comment by merriemcgaw** [comment]
Not deterministic, and fails every day or two. We should disable in 5.0 and get it fixed for 6.0p1

**Comment by merriemcgaw** [comment]
@vladimir-krestov this is pretty important. Can you focus on it next?

**Comment by vladimir-krestov** [comment]
@merriemcgaw I couldn't reproduce this issue locally using VisualStudio and cmd. I didn't catch this bug running the single test or all tests.
Should I add [Skip] attribute to this test for .Net 5.0 and try to fix it in the future in .Net 6.0 scope?

**Comment by RussKie** [comment]
The issue is intermittent, yet persistent. We had a number of builds failed citing this tests as a culprit.
Skip it in release/5.0 branch.

**Comment by merriemcgaw** [comment]
@RussKie that seems reasonable to me.

**Comment by RussKie** [comment]
The test has been re-enabled, several test run have passed.
If the issue resurfaces, will re-open.
